pkg/rotator/rotator.go

```diff
@@ -521,13 +568,53 @@ func (r *ReconcileVWH) Reconcile(request reconcile.Request) (reconcile.Result, e
 		if err := r.client.Update(r.ctx, vwh); err != nil {
 			return reconcile.Result{Requeue: true}, err
 		}
+
+		// Ensure certs on CRDs.
+		if result, err := r.ensureCRDCerts(artifacts.CertPEM); err != nil {
+			return result, err
+		}
+
+		if !*r.wasCAInjected {
```

Comment:
updating booleans is not thread-safe in golang. This either needs an atomic wrapper or it needs a mutex. Atomic wrapper seems simpler IMO.

Reply:
sg, will update.